cmd/ingress/ingress.go

```diff
@@ -144,6 +144,7 @@ the apisix cluster and others are created`,
 	cmd.PersistentFlags().DurationVar(&cfg.Kubernetes.ResyncInterval.Duration, "resync-interval", time.Minute, "the controller resync (with Kubernetes) interval, the minimum resync interval is 30s")
 	cmd.PersistentFlags().StringSliceVar(&cfg.Kubernetes.AppNamespaces, "app-namespace", []string{config.NamespaceAll}, "namespaces that controller will watch for resources")
 	cmd.PersistentFlags().StringVar(&cfg.Kubernetes.IngressClass, "ingress-class", config.IngressClass, "the class of an Ingress object is set using the field IngressClassName in Kubernetes clusters version v1.18.0 or higher or the annotation \"kubernetes.io/ingress.class\" (deprecated)")
+	cmd.PersistentFlags().StringVar(&cfg.Kubernetes.ApisixRouteClass, "apisix-route-class", config.ApisixRouteClass, "the class of an ApisixRoute object is set using the field ApisixRouteClassName in Kubernetes")
```

Comment:
We may just use `IngressClass` field to manage all resources.

Reply:
Ok. I will fix it later.